### ISSUE TITLE
Combine new content directory into Giza runtime

### ISSUE THREAD

**Issue by bedeho** [Joystream/joystream#2890]
The following two branches have new important features we are best served merging so that its part of `Olympia` runtime, as it unblocks release of these features in Atlas when convenient:

- **Video NFTs:** https://github.com/Joystream/joystream/tree/post_giza_content_dir
- **Scalable channel payouts + video comments &reactions:** https://github.com/Joystream/joystream/tree/new_content_features

There is also unmerged/unreviewed merge on channel forums (#2525), but its not clear we should go down that path, so we leave it out for now.

The best way to get this into `Olympia` is to first merge `Giza` work into this work (obviously not the other way around, because we should not touch a `Giza` release itself), and then later merge all of that work into `Olympia`, which I think @shamil-gadelshin has the intention to do.

**Comment by dobertRowneySr** [comment]
@bedeho video comments and scalable payments are not merged into `new_content_features`. Each feature has his own branch however

**Comment by dmtrjsg** [comment]
Duplicate of 3024
